models/modelCreator.py

```diff
@@ -0,0 +1,99 @@
+"""
```

Comment:
The second part of the modelMaker refactoring - this is the class for our model handling.